packages/stable/src/api/sequences/sequencesApi.ts

```diff
@@ -39,6 +39,16 @@ export class SequencesAPI extends BaseResourceAPI<Sequence> {
     );
   }
 
+  /**
+   * @hidden
+   */
+  protected getDateProps() {
+    return this.pickDateProps(
```

Comment:
This interface is unintuitive to me. What does this mean 🤔 ?

Reply:
`this.pickDateProps` is typed, so that should help a bit
but you can try to suggest something else
we can make `['items']` as a default path I guess

Reply:
After we talked yesterday then I understand the approach due to typing